app/src/main/java/com/github/kulebin/myfoursquareapp/dataSource/FoursquareDataSource.java

```diff
@@ -12,24 +12,11 @@
 
     @Override
     public void fetchVenueList(final IOnResultCallback<List<Venue>> pOnResultCallback) {
-        mIDataLoader.loadData(Api.getVenuesTrendingUrl(), pOnResultCallback);
+        mIDataLoader.loadData(Api.getVenuesTrendingUrl(), pOnResultCallback, ProcessorType.VENUES_LIST);
```

Comment:
It's better to arrange arguments in other way.
You will often implement callback when you call loadData, so it's more readable to have simple args first, and implementations after
For example - url, processor, callback

Doesn't requare fix, just for your information